tests/qos/test_pfc_counters.py

```diff
@@ -182,40 +182,40 @@ def run_test(fanouthosts, duthost, conn_graph_facts, fanout_graph_facts, leaf_fa
                     assert pfc_rx[intf]['Rx'][i] == '0'
 
 
-def test_pfc_pause(fanouthosts, duthosts, rand_one_dut_hostname,
+def test_pfc_pause(fanouthosts, duthosts, rand_one_tgen_dut_hostname,
```

Comment:
Is tgen designed for ixia test only? @sdszhang

Reply:
it was designed to be used for ixia test only.  But should be fine to be used here. Or, we can rename it to make it more generic?

Reply:
Yes, please do so.
@sdszhang please draft a separate PR for the renaming, I looked into the implementation, it could be used more generic